torch/_refs/__init__.py

```diff
@@ -2523,6 +2524,13 @@ def vstack(tensors: TensorSequenceType) -> TensorLikeType:
     return cat(aligned_tensors, 0)
 
 
+# This not an op, just a method on tensor, so we don't register decomp
```

Comment:
What do you mean by "we don't register decomp"?

Reply:
Please ignore this, I was confused about something. I'll remove this comment.